python/tvm/relay/frontend/tflite.py

```diff
@@ -1991,20 +1994,29 @@ def convert_conv(self, op, conv_type):
         weight_tensor_type_str = self.get_tensor_type_str(weight_tensor_type)
 
         in_expr = self.get_expr(input_tensor_idx)
-        weight_value = self.get_tensor_value(weight_tensor)
-
-        # TFLite kernel layout:
-        # convolution:
-        # OC KH KW IC, we require KH KW IC OC (HWIO)
-        # depthwise convolution:
-        # 1 KH KW C(input_c * depth_multiplier), we require
-        # KH KW IC M (depth_multiplier) (HWOI)
-        if is_depthwise_conv:
-            weight_value = weight_value.reshape(kernel_h, kernel_w, input_c, depth_multiplier)
+
+        if self.has_expr(weight_tensor.tensor_idx):
+            weight_expr = self.get_expr(weight_tensor.tensor_idx)
+            if is_depthwise_conv:
+                weight_expr = _op.reshape(
+                    weight_expr, (kernel_h, kernel_w, input_c, depth_multiplier)
+                )
+            else:
+                weight_expr = _op.transpose(weight_expr, axes=(1, 2, 3, 0))
```

Comment:
Few questions about this: 
* is this related to `float16` support? 
* is your test hitting this code?
* could you write some comment to explain when this branch is taken and why?

Same questions hold for the bias later

Reply:
Oh, ok now I see, thanks for the explanation! When you refer to a Quantize op, you mean a float32->float16 quantization, right? Also, could you write this in a comment? In this way everyone that will read this can understand the logic

Reply:
Oh, my bad. It should be Dequantize instead of Quantize. And yes I'll write that in the comment. Thanks.

Reply:
Hi @euntaik , what I meant was to write it as comment into the code (before the `if` statement). It is so that when people go through the code they will know what is going on.